<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Table</class>
 <widget class="QWidget" name="Table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QPushButton" name="pushButton_Home3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;

</string>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>50</y>
     <width>921</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_Ajouter_Table">
    <widget class="QLabel" name="bck_pg_ajtr">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>921</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/people-playing-ludo-gamejpg.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>391</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-radius:60px;
background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/img/plus-circle-solid (1).svg);
border: 3px solid #fa9746;
border-radius:50%;
padding:10px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom_table_ajouter">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid #fa9746;
background-color:transparent;
color:grey;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Nom </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter_table">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:2px solid #fa9746;
border-radius:25px;
</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:none;
background-color:transparent;
color:grey;</string>
      </property>
      <property name="text">
       <string>Type :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Ajouter_Table">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:1px solid  #fa9746;
border- radius:3px;
padding: 1px 18px 1px 3px;
background-color:transparent;
</string>
      </property>
     </widget>
     <zorder>lineEdit_nom_table_ajouter</zorder>
     <zorder>label_5</zorder>
     <zorder>pushButton_ajouter_table</zorder>
     <zorder>label</zorder>
     <zorder>comboBox_Ajouter_Table</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="page_Afficher_Table">
    <widget class="QLabel" name="bck_pg2_ajtr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>921</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableView" name="tableEvent">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>621</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_Modifier_Table">
    <widget class="QLabel" name="bck_pg3_ajtr">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>921</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/group-therapy-illustration.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>391</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
border-radius:60px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/img/edit-solid (1).svg);
border: 3px solid #fa9746;
border-radius:50%;
padding:10px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom_mod_table">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid #fa9746;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>280</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:2px solid #fa9746;
border-radius:25px;
</string>
      </property>
      <property name="text">
       <string>Enregistrer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Modifier_Table">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:1px solid  #fa9746;
border- radius:3px;
padding: 1px 18px 1px 3px;
color:grey;
background-color:transparent;

</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:none;
background-color:transparent;
color:grey;</string>
      </property>
      <property name="text">
       <string>Type :</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_Supprimer_Table">
    <widget class="QLabel" name="bck_pg4_ajtr">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>921</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/puzzled-woman-choosing-two-buttons-pushing-blue-one.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>371</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
border-radius:60px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="lineEdit_nom_table_suppr">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid #c25711;
background-color:transparent;
color:grey;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Nom </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_suppr_table">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:2px solid #c25711;
border-radius:25px;
</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/img/trash-solid.svg);
border: 3px solid #c25711;
border-radius:50%;
padding:10px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>489</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_Ajouter_Table">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Afficher_Table">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Modifier_Table">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Supprimer_Table">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
